psalm-baseline.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.6.4@97fe86c4e158b5a57c5150aa5055c38b5a809aab">
```

Comment:
You can read more about baseline [here](https://psalm.dev/docs/running_psalm/dealing_with_code_issues/#using-a-baseline-file) if you don't know what it is.

I have a version locally with all errors fixed. It will be the subject of other PRs.